### PR TITLE
KAA-1627: added Linux guide for flashing CC32XX

### PR DESCRIPTION
**Pull request by yverzun**

### CODE REVIEW

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md [hunk 1]
```diff
@@ -170,6 +170,17 @@ Jumpers on the CC3200 board should be connected as shown below.
 
 ![Jumpers](attach/jumpers_debug_mode_400.png)
 
-To run an application, remove SOP2 and J8 jumpers, and then connect J2 and J3 jumpers.
+Then install UniFlash for [Linux](http://software-dl.ti.com/dsps/forms/self_cert_export.html?prod_no=uniflash_3.4.1.00012_linux.tar.gz&ref_url=http://software-dl.ti.com/ccs/esd/uniflash/) or [Windows](http://software-dl.ti.com/dsps/forms/self_cert_export.html?prod_no=uniflash_3.4.1.00012_win32.zip&ref_url=http://software-dl.ti.com/ccs/esd/uniflash/).
```
Comment (ohamov) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md]: The Item is called **Flashing (Windows only)**, but you're describing both linux and windows flashing

Comment (ohamov) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md]: Omit 'Then'

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md [hunk 2]
```diff
@@ -170,6 +170,17 @@ Jumpers on the CC3200 board should be connected as shown below.
...
+
+NOTE: UniFlash v3.4.1 or later is needed. Starting from v4.0.0 UniFlash doesn't support CC32XX/CC31XX.
+
+If you are using Linux you'll need to permit execution of installation file (name of file may differ from your's):
```
Comment (ohamov) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md]: `s/you'll/you`

Comment (ohamov) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md]: `s/installation file/the installation file`
`s/name of file/the name of the file`
Of course, the name of the file differs from the user's one. Though It is not clear whether the name of user's installation file can be a different one. Rephrase this.

**Review by ohamov — CHANGES_REQUESTED** [review]

**Review by ohamov — APPROVED** [review]

**Review by RostakaGmfun — APPROVED** [review]
LGTM

**Pull request merged**